app/torii-providers/arcgis-oauth-bearer.js

```diff
@@ -9,6 +9,8 @@
  import ENV from '../config/environment';
  import Ember from 'ember';
 
+ import request from 'arcgis-rest-request';
```

Comment:
this line now throws:

> loader.js:247 Uncaught Error: Could not find module `arcgis-rest-request` imported from `dummy/torii-providers/arcgis-oauth-bearer`

i don't see any difference between this syntax and what is used in [`ember-cli-cedar`](https://github.com/Esri/ember-cli-cedar/blob/3f2a287e57c42571d8157d750a68a2b2ab143216/addon/components/cedar-chart/component.js#L4) 

the alternatives below resulted in similar errors

```js
import arcgisRest from 'arcgis-rest-request';
import request from '@esri/arcgis-rest-request';
```